Khan/perseus
# Support visible/aria label for the Expression widget

## Summary:

Adds two new options for the Expression widget via the ExpressionEditor:

- visibleLabel
- ariaLabel

Caitlyn wanted these to be separate so that they could read differently. Her example is if the label should be "Number in cm": a screen reader might read that as "Number in C M" whereas we want it to read "Number in centimeters." So this supports that case.

Issue: LEMS-2081

## Test plan:

Looking at the labels:
- Go to a page that has an Expression with label ([like this page](https://prod-znd-240724-23739-eb7fb49.khanacademy.org/internal-courses/test-everything/test-everything-1/expression/a/expression-widget-for-level-access))
- Note there is a visible label
- Note that when you click the visible label, it focuses the MathQuill input
- Note that the MathQuill input (a textarea) has an aria label

Editing the labels:
- Go to an editing page that has an Expression widget ([like this page](https://prod-znd-240724-23739-eb7fb49.khanacademy.org/devadmin/content/articles/expression-widget-for-level-access/x24be113497bc5d30))
- Note in the dropdown for the Expression options there are two new fields for visible and aria labels
- Edit the labels, publish, wait for release, and note the labels are saved
